scripts/solve_network.py

```diff
@@ -1436,6 +1437,16 @@ def solve_network(
         limit_max_growth=snakemake.params.get("sector", {}).get("limit_max_growth"),
     )
 
+    if ember_hotfix:
+        n.remove("Store", "EU gas Store")
+        n.remove("Store", "EU coal Store")
+        n.remove("Store", "EU lignite Store")
+        
+        n.links.loc[['IT0 0 CCGT', 'IT4 0 CCGT'], 'efficiency'] = 0.475
+        
+        n.links.loc[['DE0 0 CCGT', 'DE0 1 CCGT'], 'p_nom'] *= 0.69  
```

Comment:
what if `EU gas Store`, `EU coal Store`, `EU lignite Store` don't exist in `Stores`?

the efficiencies should be updated in the `custom_powerplants.csv` file, not manually in the script